### DISCUSSION TITLE
I really need help, I am thinking if I should use Nuxt or Pure Vue.

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#9212] [Questions (Nuxt 2)] [answered]
So I created a sample Nuxt To test its SEO first by creating my own blog website before before creating the project.

 So that if it's working in a search engine like google or duck duck go, Like for example searching in google's `site: sample.ml` 

I have set up the domain, added sitemap, set up SEO.
waited for days, because maybe it takes time for the crawlers to crawl the website.

But when I type my blog site created in nuxt  `site: brojenuel-blog.ml` on google its not working, its not showing anything.

But my portfolio website created using Vue-CLI is working in google search: `site: jenuelganawed.ml` or just `jenuel ganawed` in google.com

**Comment by TheAlexLichter** [comment, marked as answer]
SEO is a bit more than using Nuxt (but I can assure it is a great step in the right direction) 👍🏻

At first: Have you registered your site in the Google Search Console? If not, better do that to make sure it will be crawled! Otherwise Google has to "find out" on it's own about your site, e.g. through backlinks from other sites.

Maybe [this talk](https://www.youtube.com/watch?v=MGvtkh2asv4) could be of interest ☺️

**Reply by JenuelDev** [reply]
Good day, 
I added it in the google search console, and it says 
"Processing data, please check again in a day or so", 

maybe I'll be checking it later again.

and I will come back here later. Thanks thanks 👍👍
